runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -100,14 +101,16 @@ public Layout getCurrentLayout() {
                 } else {
                     throw re;
```

Comment:
Should we wrap this as an Unrecoverable Exception too?

Reply:
Yes, updated